### DISCUSSION TITLE
Menu widget: mobile dropdown doesn't close when clicking on a link that leads to an anchor tag

### DISCUSSION THREAD

**Discussion by ahmedzeidan-ul** [elementor/elementor#23233] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Menu widget: Close the mobile drop-down menu when the hyperlink is an anchor tag.

### Describe the solution you'd like

Menu widget: On mobile, when the user clicks on a menu item that has an anchor tag, the mobile dropdown menu should close automatically.

### Additional context

This feature is available in the old WordPress nav widget but not available in the new Menu Widget (mega menu).

**Comment by BillGitUser** [comment]
This problem still exists today 9/22/2023 still.  And to clarify, its not if the menu item has an anchor tag...its if the menu item has an anchor tag AND the anchor is on the current page.

**Reply by ahmedzeidan-ul** [reply]
@BillGitUser correct, it is if the anchor tag is on the same page. As per the Elementor team, this is not a bug and it is a new feature to be added to the new Menu Widget. We hope to hear back from them on this improvement for the widget so that we can replace the WordPress Nav widget with the new Menu widget. @nicholaszein

**Reply by BillGitUser** [reply]
@ahmedzeidan-ul Thanks for your reply. Can you clarify what you mean by the Wordpress Nav Widget and the new Menu widget.  My menu widget is "Wordpress Menu".  Is that what you mean by "Wordpress Nav"?  And is Elementor coming up with their own Menu widget to replace this?  I see a Navigation Menu Lite" widget as an existing option.  Did you mean that widget with the update we are asking for?

**Reply by ahmedzeidan-ul** [reply]
@BillGitUser correct, there is a new Menu widget that you might need to activate in the experiment. This new feature request is related to the new Elementor Menu widget and not the WordPress Menu.
